cmd/console/main.go

```diff
@@ -189,6 +189,12 @@ func printResult(summary model.Summary) error {
 			fmt.Printf("\t%s:%d\n", f.FileName, f.Line)
 		}
 	}
+	fmt.Printf("Kicks Summary\n")
```

Comment:
The title could be: **Results Summary**
I would switch to the following order:
- HIGH
- MEDIUM
- LOW
- INFO
- TOTAL

Reply:
Added sugestions